src/pages/AddStoryPage/AddStoryPage.tsx

```diff
@@ -65,7 +65,7 @@ export const AddStoryPage = ({
       .then((data) => {
         // Success! Move on to the full edit form
         history.push(
-          `/${feed?.name}/prospects/article/edit-and-approve/${data.data?.prospect.id}/`,
+          `/${feed?.name}/prospects/${data.data?.prospect.id}/edit-and-approve/`,
```

Comment:
a little concerned about the `data.data?.` part here - if `data.data` is empty, we probably shouldn't push to the history, right?

Reply:
Good point, I've looked up Apollo types and it can potentially be null or undefined.